python/cugraph/dask/opg_pagerank/pagerank.py

```diff
@@ -20,30 +20,54 @@
 from cugraph.opg.link_analysis import mg_pagerank_wrapper as mg_pagerank
 
 
-def common_func(sID, data, local_data):
-    print("Dataframe: ", data)
-    print("local data: ", local_data)
+def common_func(sID, data, local_data, alpha, max_iter, tol, personalization, nstart):
     sessionstate = worker_state(sID)
-    mg_pagerank.mg_pagerank(data[0], local_data, sessionstate['handle'])
-    return 1
+    return mg_pagerank.mg_pagerank(data[0],
+                                   local_data,
+                                   sessionstate['handle'],
+                                   alpha,
+                                   max_iter,
+                                   tol,
+                                   personalization,
+                                   nstart)
 
 
-def pagerank(input_graph):
-    print("INSIDE DASK PAGERANK")
+
+def pagerank(input_graph,
```

Comment:
We should probably have doc here and update https://github.com/rapidsai/cugraph/blob/branch-0.15/docs/source/dask-cugraph.rst.tmp

Reply:
updated